eng/pipelines/doc-index.yml

```diff
@@ -8,25 +8,23 @@ jobs:
         inputs:
           versionSpec: '3.6'
 
-      - template: /eng/common/pipelines/templates/steps/replace-relative-links.yml
-        parameters:
-          TargetFolder: '.'
-          RootFolder: '.'
-          BuildSHA: $(Build.SourceVersion)
-          RepoId: 'Azure/azure-sdk-for-cpp'
-
       - pwsh: |
           Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
           -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
+          echo "##vso[task.setvariable variable=docfxPath;isOutput=true]$(Build.SourcesDirectory)/docfx/docfx.exe"
         workingDirectory: $(Build.SourcesDirectory)
         displayName: Download and Extract DocFX
-
-      - pwsh: >-
-          $(Build.SourcesDirectory)/eng/docs/index/Generate-DocIndex.ps1
-          -RepoRoot $(Build.SourcesDirectory)
-          -DocGenDir "$(Build.SourcesDirectory)/eng/docs/index"
-          -verbose
-        displayName: 'Generate Doc Index'
+        name: setupDocfxTool
```

Comment:
What is the name for here?

Reply:
I have tested over the variable $(docfxToolPath), the only way to make it work is to use the step name to call it.

`$(setupDocfxTool.docfxPath)`

Reply:
The variable shared across steps.

Reply:
Oh... that is because you set the "isOutput=true" for this use case that isn't needed but also don't really hurt anything.